joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/AstForExpressionCreator.scala

```diff
@@ -8,13 +8,13 @@ import io.shiftleft.codepropertygraph.generated.nodes.NewCall
 import io.shiftleft.codepropertygraph.generated.{DispatchTypes, Operators}
 
 trait AstForExpressionCreator { this: AstCreator =>
-  def astForExpression(expr: ParserNodeInfo): Seq[Ast] = {
+  def astsForExpression(expr: ParserNodeInfo): Seq[Ast] = {
     expr.node match {
       case BinaryExpr => astForBinaryExpr(expr)
       case StarExpr   => astForStarExpr(expr)
       case UnaryExpr  => astForUnaryExpr(expr)
-      case ParenExpr  => astForExpression(createParserNodeInfo(expr.json(ParserKeys.X)))
-      case _          => Seq()
+      case ParenExpr  => astsForExpression(createParserNodeInfo(expr.json(ParserKeys.X)))
+      case _          => Seq(Ast())
```

Comment:
The following should probably be fine? If not, it's not an issue really
```suggestion
      case _          => Seq.empty[Ast]
```

Reply:
@DavidBakerEffendi One observation is if we use `Seq()` or `Seq.empty[Ast]` an empty Block node is created, which is not required

Reply:
That is interesting, lol. But cool will merge this